sdg/components.py

```diff
@@ -355,6 +357,8 @@ def get_git_host(repo_url):
             pipeline=pipeline,
             chunk_word_count=1000,
             server_ctx_size=4096,
+            batch_size=sdg_batch_size,
+            num_cpus=sdg_num_workers,
```

Comment:
Can we follow the same naming pattern for the parameters? It makes it easier to match the parameter value passing throughout the functions